hw_9/functional.py

```diff
@@ -0,0 +1,97 @@
+#!/usr/bin/env python
+# coding: utf-8
+
+# In[115]:
+
+
+import sys
+
+
+# In[59]:
+
+
+# sequential_map
+def sequential_map(*args):
+    container = args[-1]
+    for i in args[:len(args) - 1]:
+        container = list(map(i, container))
+    return(container)
```

Comment:
1. Можно чуть оптимальнее достать функции и контейнер, чтобы потом не пришлось делать срезы
2. `i` - очень плохое имя для функции
```suggestion
    *functions, container = args
    for func in functions:
        container = map(func, container)
    return list(container)
```